client/src/components/Heading1.js

```diff
@@ -0,0 +1,7 @@
+import style from "styled-components";
```

Comment:
I can see you are using styled components, is the package installed? Is it possible to use just css to achieve this?

Reply:
Hi yes it is.. I struggled with using just css. Will try again though

Reply:
It's ok, if it is installed then it is not an issue